.ci/docker-ci/arch/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM archlinux:base-20220220.0.48372
+
+LABEL maintainer="mail@sobolevn.me"
+LABEL vendor="git-secret team"
+
+RUN pacman -Syu --needed --noconfirm \
```

Comment:
What do we need to do to package `git-secret` for Arch? (I've last used it like 15 years ago)

Reply:
I don't know! I guess we'll have to look at some examples and/or read up on Arch packaging a little. Perhaps someone can assist

Reply:
https://wiki.archlinux.org/title/creating_packages